packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.3.4
 
-* Adds `countryCode` API. 
```

Comment:
You'd need to bypass checks to fix the things in this package without a release; I would just revert them and we can clean them up as drive-bys later.

Reply:
Oh, you already bypassed the check. That works too.

Reply:
ah ok

Reply:
I saw your comment right after. This version has no android changes and the version check override is removed.